beesdoo_product/readme/DESCRIPTION.rst

```diff
@@ -1,2 +1,6 @@
 Modification of product module for the needs of beescoop
-- SOOO5 - Ajout de label bio/ethique/provenance
+- SOOO5 - Adds the label bio/ethique/provenance
+- Add a 'Suggested Price' field on products, and a 'Product Margin' field on Partners (Vendors) and Product Categories. The first margin is used if set, otherwise the second margin (which has a default value) is used.
```

Comment:
Just add a note about the assumption that we made for this module. We suppose that each product has only one supplier and that same product coming from different suppliers should be duplicated.